types/relay-runtime/relay-runtime-tests.tsx

```diff
@@ -54,8 +54,9 @@ function handlerProvider(handle: any) {
         // Augment (or remove from) this list:
         case 'connection':
             return ConnectionHandler;
-        case 'viewer':
-            return ViewerHandler;
+        // case 'viewer':
+        //     // ViewerHandler is special-cased and does not have an `update` method
+        //     return ViewerHandler;
```

Comment:
Should this comment stay?

Reply:
I'm not sure, because the docs (https://github.com/facebook/relay/blob/937579ba88d885205327399504e1b17d1f8b0dc3/website/versioned_docs/version-v5.0.0/Modern-RelayEnvironment.md#adding-a-handlerprovider) of relay-runtime still call out `ViewerHandle`, but it definitely wouldn't execute properly given that it has no `.update` method, thus my comment of "ViewerHandler is special-cased and does not have an `update` method".

Reply:
Ok. We'll see if anyone actually using this runs into an issue and can help fix.